sbc_compassion/tools/import_letter_functions.py

```diff
@@ -242,12 +383,13 @@ def _save_img(MULTIPAGE, file_, img):
     return data
 
 
-def _find_template(env, img, line_vals):
+def _find_template(env, img, line_vals, test):
     """
     Use pattern recognition to detect which template corresponds to img.
     :param env env: Odoo variable env
     :param img: Image to analyze
     :param dict line_vals: Dictonnary containing the data for a line
+    :param bool test: Save the image of all the template detections
```

Comment:
This is really a bad choice for variable name if it does what it says in this comment.